assets/js/app.js

```diff
@@ -18,8 +18,8 @@
 // Include phoenix_html to handle method=PUT/DELETE in forms and buttons.
 import "phoenix_html"
 // Establish Phoenix Socket and LiveView configuration.
-import {Socket} from "phoenix"
-import {LiveSocket} from "phoenix_live_view"
+import { Socket } from "phoenix"
```

Comment:
Please leave this as it is. `{Socket}`

Are these modifications due to some tool?

Reply:
I don't remember directly fixing the relevant part, so I think it might be due to the impact of mix format.

Reply:
The `mix format` is an Elixir code formatter and should not touch JavaScript code.

I suspect that you have a code formatter tool installed for JavaScript or Node.js.

There is nothing wrong with using a code formatter tool, but all project members need to use the same tool with the same settings.